sympy/core/relational.py

```diff
@@ -381,12 +382,21 @@ class Equality(Relational):
 
     is_Equality = True
 
-    def __new__(cls, lhs, rhs=0, **options):
+    def __new__(cls, lhs, rhs=None, **options):
         from sympy.core.add import Add
         from sympy.core.logic import fuzzy_bool
         from sympy.core.expr import _n2
         from sympy.simplify.simplify import clear_coefficients
 
+        if rhs is None:
+            SymPyDeprecationWarning(
+                feature="Eq(expr) with rhs default to 0",
+                useinstead="Eq(expr, 0)",
+                issue=16587,
```

Comment:
This is the issue tagged with deprecation removal. Should it actually be the issue that explains why this feature has been changed?

Reply:
I think the tag is correct. The issue tagged with deprecation removal should give a brief explanation of why the feature has been changed.